packages/account/src/Sections/Profile/LanguageSettings/language-settings.tsx

```diff
@@ -1,78 +1,51 @@
 import React from 'react';
+import { Redirect } from 'react-router-dom';
 import { useTranslation } from 'react-i18next';
-import { Button, DesktopWrapper } from '@deriv/components';
 import { localize, getAllowedLanguages } from '@deriv/translations';
+import { isMobile, routes } from '@deriv/shared';
 import { connect } from 'Stores/connect';
 import FormSubHeader from 'Components/form-sub-header';
-import TCoreStore from '../../../Stores';
-import { Formik, FormikHandlers, FormikHelpers, FormikValues } from 'formik';
-import FormFooter from 'Components/form-footer';
 import LanguageRadioButton from 'Components/language-settings';
+import TCoreStore from '../../../Stores';
 
 type TLanguageSettings = {
     current_language: string;
     changeSelectedLanguage: (lang: string) => void;
-    isCurrentLanguage: (lang: string) => boolean;
 };
 
-const LanguageSettings = ({ changeSelectedLanguage, current_language, isCurrentLanguage }: TLanguageSettings) => {
+const LanguageSettings = ({ changeSelectedLanguage, current_language }: TLanguageSettings) => {
     const { i18n } = useTranslation();
+
+    if (window.location.pathname === routes.languages && isMobile()) {
```

Comment:
prevent loading language-settings page for responsive.

Reply:
@yauheni-deriv This doesn't seem right, We should have this check where we are rendering the component so it never renders for that case 🤔 Or maybe I'm missing something 🤔

Reply:
Actually this component is not supposed to render, because we are returning just div for this route (you can see it in this PR in toggle-mobile-drawer on line 37-50).
this redirect here implemented just for such case, when client enters exact url address in mobile. so we prevent opening this page, because for responsive we propose clients to change language only with hamburger menu, but this menu can be opened with two links